src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -458,6 +458,23 @@ public void testReadsThatConsumeZeroReferenceReads() throws Exception {
         runCommandLine(args);
     }
 
+    // test fix for https://github.com/broadinstitute/gatk/issues/3466
+    @Test
+    public void testCompletelyClippedReadNearStartOfContig() throws Exception {
+        final File testCaseFilesDir = new File(TEST_FILES_DIR, "issue3466_gatk_cigar_error");
```

Comment:
You should probably move these test files into some other folder, like the  HaplotypeCaller folder rather than making its own folder for this one commit

Reply:
I'd rather keep them separated, since they are very specific to this bug.